src/humanize/number.py

```diff
@@ -119,23 +119,26 @@ def intcomma(value, ndigits=None):
         return intcomma(new)
 
 
-powers = [10 ** x for x in (6, 9, 12, 15, 18, 21, 24, 27, 30, 33, 100)]
-human_powers = (
-    N_("million"),
-    N_("billion"),
-    N_("trillion"),
-    N_("quadrillion"),
-    N_("quintillion"),
-    N_("sextillion"),
-    N_("septillion"),
-    N_("octillion"),
-    N_("nonillion"),
-    N_("decillion"),
-    N_("googol"),
+default_conversions = [
```

Comment:
Is `default_conversions` an internal thing? Should we name it `_ default_conversions`?

Reply:
`default_conversions` is now where `powers` and `human_powers` are defined. If `powers` and `human_powers` were also internal, then maybe `default_conversions` should also be internal?

Reply:
The existing `powers` and `human_powers` are already "public", so we shouldn't change those to `_powers` and `_human_powers` in case people are using them.

But do you think people would or should be able to change the default conversions?

Reply:
Changing the defaults might not make sense, but people may want to expand them, eg.

```python

from humanize.numbers import intword, default_conversions

intword(1023, conversions=default_conversions.append((3,"Thousands"))

```

But I'm not sure how useful that would be in practice.

Reply:
Okay, let's keep it as `default_conversions`